### PR TITLE
Bg 22 create log display

### PR DESCRIPTION
**Pull request by romelu**

### CODE REVIEW

File: code/css/theme-sand.css [hunk 1]
```diff
@@ -7,11 +9,63 @@ h1 {
...
+  padding: 5px;
+  font-size: 20px;
+  border: none;
+}
+
+/*  box for entire view - needed to position the content */
+
+.rcalc-container-view {
+  text-align: center;
+  background-color: #e2d9b5;
```
Comment (palomino7) [inline code/css/theme-sand.css:53]: Just small letters or big letters. Unify it ... #e2d9b5 ... #FFFFFF

File: code/js/constant/string-attribute-element-constant.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const STRING_ATTRIBUTE_ELEMENT_CONST = {
```
Comment (palomino7) [inline code/js/constant/string-attribute-element-constant.js]: HTML_ELEMENT_ATTRIBUTE_CONST would be better.

File: code/js/rcalc-log.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  var parentDiv = document.getElementById(HTML_ID_CONST.ID_LOG_DISPLAY);
+  var paragraph = document.createElement(STRING_ATTRIBUTE_ELEMENT_CONST.ELEMENT_P);
+  var text;
+  var nextToFirstChild;
+  paragraph.style.backgroundColor = HTML_CSS_CONST.COLOR_LIGHT_GREY;
+  paragraph.setAttribute(STRING_ATTRIBUTE_ELEMENT_CONST.ATTRIBUTE_CLASS, HTML_CLASS_CONST.RCALC_LOG_DISPLAY_LINE);
+  text = document.createTextNode(leftSideOfEqualSignGlobal + STRING_UTILITY_CONST.EQUALS_SIGN + resultGlobal);
+  paragraph.appendChild(text);
+  parentDiv.insertBefore(paragraph, parentDiv.firstChild);
+  nextToFirstChild = parentDiv.firstChild.nextSibling;
```
Comment (palomino7) [inline code/js/rcalc-log.js]: nextToFirstChild = parentDiv.firstChild.nextSibling;
  parentDiv.removeChild(nextToFirstChild);
... do it by one line without var nextToFirstChild

File: code/js/rcalc-log.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  parentDiv.removeChild(nextToFirstChild);
+  changeLineColor(paragraph);
+}
+
+function setDisplayLineErrorCase() {
+  var parentDiv = document.getElementById(HTML_ID_CONST.ID_LOG_DISPLAY);
+  var paragraph = document.createElement(STRING_ATTRIBUTE_ELEMENT_CONST.ELEMENT_P);
+  var text;
```
Comment (palomino7) [inline code/js/rcalc-log.js]: just cosmetic ... put one empty line after declaration of vars.

File: code/css/theme-sand.css (hunk 1 above)
Comment (romelu) [reply code/css/theme-sand.css:53]: unified... just small letters used

File: code/js/rcalc-log.js (hunk 4 above)
Comment (romelu) [reply code/js/rcalc-log.js]: empty line added after declaration of vars

File: code/js/constant/string-attribute-element-constant.js (hunk 2 above)
Comment (romelu) [reply code/js/constant/string-attribute-element-constant.js]: file renamed according your advice

File: code/js/rcalc-log.js (hunk 3 above)
Comment (romelu) [reply code/js/rcalc-log.js]: only one line like this parentDiv.removeChild(parentDiv.firstChild.nextSibling);
var nextToFirstChild removed... no needed

**Pull request merged**